catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableConfigTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.migratable.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CatalogMigratableConfigTest {
+
+    private CatalogMigratableConfig config;
+
+    @Before
+    public void setup() throws Exception {
+        config = new CatalogMigratableConfig();
+    }
+
+    @Test
+    public void testExportQueryPageSizeValid() throws Exception {
+        config.setExportQueryPageSize(70000);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testExportQueryPageSizeTooLarge() throws Exception {
+        config.setExportQueryPageSize(100001);
```

Comment:
I don't like the magic values in this class, even though they are unit tests. It'd be nice if you could at least make these values consts in the class so it's clearer to a future developer that this value is larger than the max allowed.